lib/Support/Random.cpp

```diff
@@ -15,12 +15,27 @@
  */
 
 #include "glow/Support/Random.h"
+#include "llvm/Support/CommandLine.h"
 
 #include <cassert>
-#include <random>
+
+static llvm::cl::opt<glow::PseudoRNG::result_type>
+    pseudoRandomSeed("pseudo-random-seed",
+                     llvm::cl::desc("Seed for pseudo-random numbers"),
+                     llvm::cl::init(glow::PseudoRNG::Engine::default_seed));
 
 namespace glow {
 
+PseudoRNG::PseudoRNG() : engine_(pseudoRandomSeed) {}
+
+// Constant uniform distribution used as a template in nextRand.
+// This computes the parameters once instead of on each call.
+const static std::uniform_real_distribution<> uniformReal(-1, 1);
+
+double PseudoRNG::nextRand() {
+  return std::uniform_real_distribution<double>(uniformReal.param())(engine_);
+}
+
 double nextRand() {
```

Comment:
Do we still need this method?

Reply:
No, I'll remove it in a third commit along with any lingering call sites that use it. Same for the global `nextRandInt()`.